packages/fresh/analysis_options.yaml

```diff
@@ -9,3 +9,5 @@ linter:
 analyzer:
   exclude:
     - test/.test_coverage.dart
+  errors:
+    deprecated_member_use: ignore
```

Comment:
why is this needed?

Reply:
Since I defined the Token class as depreciated, the analyze dart is causing the checks to fail. Add to make all checks pass.
As the package will have a considerable break change, should we already remove the Token class?

Reply:
yup will just publish 0.3.0 👍